LEFT_HAND:
- ACTION_NAME: 张开
  ACTION_POS:
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 10
  - 100
  - 180
  - 240
  - 245
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
RIGHT_HAND:
- ACTION_NAME: 张开
  ACTION_POS:
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 10
  - 100
  - 180
  - 240
  - 245
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
- ACTION_NAME: 握拳
  ACTION_POS:
  - 69
  - 0
  - 0
  - 0
  - 0
  - 151
  - 10
  - 100
  - 180
  - 240
  - 14
  - 255
  - 255
  - 255
  - 255
  - 109
  - 0
  - 0
  - 0
  - 0
- ACTION_NAME: '666'
  ACTION_POS:
  - 255
  - 255
  - 33
  - 255
  - 255
  - 255
  - 10
  - 100
  - 180
  - 240
  - 245
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
